evm-ds/src/precompiles/blake2.rs

```diff
@@ -200,7 +200,7 @@ fn required_gas(input: &[u8]) -> Result<u64, ExitError> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    type PrecompileResult = Result<PrecompileOutput, PrecompileFailure>;
+    type PrecompileResult = Result<(PrecompileOutput, u64), PrecompileFailure>;
```

Comment:
Why did you have to change this? Is something wrong with the `evm` dependency versioning?

@bzawisto PTAL.

Reply:
@valeryz These changes were in blake2 tests. I think someone had changed the return type but didn't update the tests. They failed to build.